### DISCUSSION TITLE
ontechhindi.com: detection

### DISCUSSION THREAD

**Discussion by preetam1409** [uBlockOrigin/uAssets#24144] [General]
ontechhindi.com: detection

orginal link - https://zipshort.net/4KYr
adblocker detected in https://ontechhindi.com/what-is-liability-insurance/

**Comment by stephenhawk8054** [comment]
For shorteners, please comment at this thread: https://github.com/uBlockOrigin/uAssets/discussions/17361

Don't open duplicated discussion

**Comment by preetam1409** [comment]
okay , but I don't find the option where to add a new issue , can you
please help

**Reply by stephenhawk8054** [reply]
> For shorteners, please comment at this thread: https://github.com/uBlockOrigin/uAssets/discussions/17361

Just comment like you are doing here, but at that thread

**Comment by preetam1409** [comment]
where to add a new issue please i cannot find the button

**Reply by stephenhawk8054** [reply]
Just comment at that thread like you are doing at this thread. What is the difference?

![image](https://github.com/uBlockOrigin/uAssets/assets/66517106/79f52a92-07ad-49f6-884b-b3fdaae633a4)

Go to here: -> https://github.com/uBlockOrigin/uAssets/discussions/17361 -> Scroll down

**Comment by preetam1409** [comment]
okay sir understood, yes I have commented as per your directions

**Reply by stephenhawk8054** [reply]
You are replying to another user's comment. You can scroll down to the bottom of the page and put your own comment there.

**Comment by preetam1409** [comment]
Thanks a lot for your guidance . Sorry for mistakes

**Comment by preetam1409** [comment]
hi I have posted in there but haven't got any response please can you check

**Reply by stephenhawk8054** [reply]
Any volunteers who can look at the issue will help investigate it.

**Reply by ghajini** [reply]
fixed https://github.com/uBlockOrigin/uAssets/commit/686ff8b84f79241b3cea1a3330c7f01fc0a415bf
